api.swagger.yaml

```diff
@@ -5611,6 +5611,82 @@ paths:
           description: The user does not have access to the requested content/comment.
         '404':
           description: No content/comment with that UUID was found.
+  '/report/{content_id}':
+    parameters:
+      - name: content_id
+        description: The Invotra UUID of content/comment.
+        in: path
+        required: true
+        schema:
+          type: string
+          format: uuid
+          example: '01234567-89ab-cdef-1234-56789abcdef0'
+    put:
+      tags:
+        - report
+      description: 'Report a piece of content/comment.'
+      responses:
+        '200':
+          description: 'Reported the content/comment successfully.'
+        '400':
+          description: 'The content/comment was not reported due to malformed JSON.'
+        '403':
+          description: 'User is not authorized to report this content/comment.'
+        '422':
+          description: 'The content/comment was not reported due to Drupal validation failure or JSON schema validation failure.'
+      security:
+        - SessionCookieHeader: []
+        - APIKeyHeader: []
+        - CognitoJwt: [ invotra/report:write ]
+    delete:
+      tags:
+        - report
+      description: 'Unreport a piece of content/comment.'
+      responses:
+        '200':
+          description: 'Report removed successfully.'
+        '400':
+          description: 'Could not unreport this content/comment due to malformed JSON.'
+        '403':
+          description: 'User is not authorized to unreport this content/comment.'
+        '422':
+          description: 'The report could not be removed due to Drupal validation failure or JSON schema validation failure.'
+      security:
+        - SessionCookieHeader: []
+        - APIKeyHeader: []
+        - CognitoJwt: [ invotra/report:write ]
+    get:
+      description: 'Check if a piece of content/comment is reported or not.'
```

Comment:
Check if a piece of content/comment is reported or not.

->

Check if a piece of content/comment has been reported or not.